setup.py

```diff
@@ -21,12 +17,17 @@
     install_requires=install_requires,
     tests_require=tests_require,
     setup_requires=[],
+    python_requires='>=2.7',
     extras_require={
         'test': tests_require,
         'all': install_requires + tests_require,
         'docs': ['sphinx'] + tests_require,
         'lint': []
     },
+    classifiers=[
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.6'
+    ],
```

Comment:
I'm not actually entirely sure what this does.  I _think_ all it does is document that it's been tested with these specific versions.